secure-app/backend/src/db/init.sql

```diff
@@ -12,34 +12,82 @@ CREATE TABLE IF NOT EXISTS Users (
 
 CREATE TABLE IF NOT EXISTS Editeurs (
     idEditeur SERIAL PRIMARY KEY,
-    nomEditeur TEXT NOT NULL
+    libelleEditeur TEXT NOT NULL,
+    exposant BOOLEAN TEXT NOT NULL,
+    distributeur BOOLEAN TEXT NOT NULL DEFAULT FALSE,
```

Comment:
I'd suggest to use clearer names for these attributes:

```suggestion
    estExposant BOOLEAN TEXT NOT NULL,
    estDistributeur BOOLEAN TEXT NOT NULL DEFAULT FALSE,
```